uber/decorators.py

```diff
@@ -81,7 +81,13 @@ def _suffix_property_check(inst, name):
 def csrf_protected(func):
     @wraps(func)
     def protected(*args, csrf_token, **kwargs):
-        check_csrf(csrf_token)
+        try:
+            check_csrf(csrf_token)
+        except AssertionError as e:
+            raise HTTPRedirect("../common/invalid?message={}", "Session Time Out. Go Back And Try Again.")
```

Comment:
This is almost painfully minor, but we don't usually use Title Caps in our errors -- writing a normal sentence here should be fine.